webrtc.go

```diff
@@ -0,0 +1,259 @@
+package nkn
```

Comment:
most of the codes in this file are the same in `github.org/nknorg/nkn/api/webrtc/webrtc.go`, can we reuse it and overwrite some methods if needed?

Reply:
Yes. Let me modify it.

Reply:
Need wait  for nkn webrtc package is released.